### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by seekanddefine** [athensresearch/athens#1062] [Operations]
### purpose
- Discord is the primary home of the Athens Research community and this post is to prompt discussion on how we can improve our community, user, and developer experience using the tool
- synchronous chat, community calls, coworking
### current state
- [Roles](https://support.discord.com/hc/en-us/articles/214836687-Role-Management-101) are not formalized and outlined anywhere
- Some Bots are in use, their individual roles and functions in our server are not outlined anywhere #1347 
### goal state
- promote collaboration, creativity, and inclusivity

**Comment by seekanddefine** [comment]
How do we best create pull requests that are actionable by the most members of our community? Is there a way we could create a better flow to github to encourage collaboration? Could we have a channel is slack where we post a new request each day as well as a project amount of time to finish? Could we see what pull requests were approved and highlight the person who did it? Or maybe a place where we gather everyone's first official contribution to Athens? Trying to think of ways to get people feeling good about jumping into Github or celebrate accomplishments.

**Reply by Limezy** [reply]
For me the starting point is not PR. Way too complicated. However, starting to have the good habit of writing a small bug report as soon as you think you discovered one, this is important. Better having more bug reports but keeping track than passive users. A small portion of these "regular" github visitors may then be interested into having a look at the commits or PR, and eventually start to contribute

**Reply by Limezy** [reply]
Or even as a first step, visit the bug reports and react to it with a 👍, say thanks when it's resolved, or double confirm with a quick comment that you are able to reproduce. All this is helpful

**Comment by seekanddefine** [comment]
How does our passion for open-source meet the business of school building? In the funnel we hope that people who join us on Discord will eventually be brave enough to hop onto Gitbhub and become a project contributor coder or non. We hope for high retention but perhaps even more importantly sustainable long-term retention in seeing early contributors sticking around to be mentors to new people because of the value they  found as a member of the community.

**Comment by seekanddefine** [comment]
How can Athens discord be intentionally crafted to be an area everyone feels safe and free to discuss ideas about Athens, notetaking strategies, feeling great about supporting open-source, and/or learning Clojure?

**Comment by seekanddefine** [comment]
What if we were the go-to community for knowledge maps/networked thought/notetaking/open-source/mentoring/coding/clojure______________?

**Comment by seekanddefine** [comment]
Should we use role colors in discord in fun ways to communicate things like hours spent contributing/time spent using Clojure/your position in the Athens learningverse?

Some 100 million people use Discord every month, that they increased 50% in less than a year, or that they raised $100 million in funding as they hit 140 million users last year.

Discord is truly a place for community and not just playing video games or swapping recipes. It is very easy to download and install and has mobile.

**Comment by jsmorabito** [comment]
What other community hubs can we take inspiration from? (not exclusive to Discord communities) 
- [Sourcecred Discord](https://discord.gg/gq8xQ8p8)? 

Other open source communities? 
- [Gitcoin Discord](https://discord.gg/cTK8d6uvEs)?

Other tools for thought communities?
- [Obsidian Discord](https://discord.gg/veuWUTm)? 
- Roam Slack?

**Reply by joelhans** [reply]
Notes on the Obsidian Discord:

- Concise and useful welcome message.
- Really, really, _really_ nice release notes. Look at the love they get for each one! It's outside the scope of this discussion, but we should be aiming for release notes like this by mid/end of July.
- I like that there's only a few channels for the product.
- We should have a place like `daily-graph` for sharing screenshots.
